sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 ## Release History
 
-### 3.22.0-beta.1 (Unreleased)
+### 3.22.0 (2022-06-08)
 #### Features Added
 * Exposed `maxDegreeOfParallelism` feature from CosmosQueryRequestOptions through application.properties flag - See [PR 28756](https://github.com/Azure/azure-sdk-for-java/pull/28756)
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
-#### Other Changes
+* Updated `azure-cosmos` to version `4.31.0`.
```

Comment:
for other SDK, the updated azure-cosmos is under `Other Changes`category, should we do the same thing here

Reply:
Fixed